extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/Json.java

```diff
@@ -0,0 +1,344 @@
+package io.quarkus.vertx.http.runtime.devmode;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Objects;
+
+/**
+ * A simple JSON string generator.
+ */
+public final class Json {
```

Comment:
I'm not going to review this since you said it's just copied from Weld :)

Reply:
Well, it's not 1:1 copy and the fact that it's coming from Weld does not mean that it needs no review ;-)

Reply:
Ah, you're killing me :P